File: Tp Soap/Q1EnJava/src/Reserve/TypeChambre.java
package Reserve;

public class TypeChambre {

	private int nbrLits, prixBase;

	public TypeChambre(int prixBase, int nbrLits) {

		this.prixBase = prixBase;
		this.nbrLits = nbrLits;
	}

	public int getPrixBase() {
		return prixBase;
	}

	public void setPrixBase(int prixBase) {
		this.prixBase = prixBase;
	}

	public int getNbrLits() {
		return nbrLits;
	}

	public void setNbrLits(int nbrLits) {
		this.nbrLits = nbrLits;
	}

}

File: Tp Soap/Q2/Q2/TypeChambre.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Q2
{
    public class TypeChambre
    {
        private float prixBase;
        private int nbrLits;

        public TypeChambre(float prixBase, int nbrLits)
        {

            this.PrixBase = prixBase;
            this.NbrLits = nbrLits;
        }

        public float PrixBase { get => prixBase; set => prixBase = value; }
        public int NbrLits { get => nbrLits; set => nbrLits = value; }
    }
}